ui/components/ui/nickname-popover/nickname-popover.component.js

```diff
@@ -8,7 +8,6 @@ import Identicon from '../identicon/identicon.component';
 import { shortenAddress } from '../../../helpers/utils/util';
 import CopyIcon from '../icon/copy-icon.component';
 import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
-import './index.scss';
```

Comment:
Import this into `ui/components/ui/ui-components.scss` instead of the present file, which will allow the scss to work correctly once we start using the component in the app